internal/providers/terraform/google/testdata/compute_address_test/compute_address_test.golden

```diff
@@ -1,39 +1,47 @@
- google_compute_address.preemptible_with_usage                                       
- └─ IP address (preemptible VM)                             730  hours         $1.46 
-                                                                                     
- google_compute_address.standard_with_usage                                          
- └─ IP address (standard VM)                                730  hours         $2.92 
-                                                                                     
- google_compute_address.static                                                       
- ├─ IP address (if used by standard VM)                     730  hours         $2.92 
- ├─ IP address (if used by preemptible VM)                  730  hours         $1.46 
- └─ IP address (if unused)                                  730  hours         $7.30 
-                                                                                     
- google_compute_address.unused_with_usage                                            
- └─ IP address (unused)                                     730  hours         $7.30 
-                                                                                     
- google_compute_global_address.default                                               
- ├─ IP address (if used by standard VM)                     730  hours         $2.92 
- ├─ IP address (if used by preemptible VM)                  730  hours         $1.46 
- └─ IP address (if unused)                                  730  hours         $7.30 
-                                                                                     
- google_compute_global_address.standard_with_usage                                   
- └─ IP address (standard VM)                                730  hours         $2.92 
-                                                                                     
- OVERALL TOTAL                                                                $49.64 
+ Name                                                      Monthly Qty  Unit              Monthly Cost 
+                                                                                                       
+ google_compute_address.invalid_usage                                                                  
+ └─ IP address (unused)                                 Monthly cost depends on usage: $0.01 per hours 
+                                                                                                       
+ google_compute_address.preemptible_static                                                             
+ └─ IP address (preemptible VM)                                    730  hours                    $1.46 
+                                                                                                       
+ google_compute_address.preemptible_with_usage                                                         
```

Comment:
Can we get rid of the usage parameter now since we can detect the purpose and attachment status?  What is the case where we still need to support the usage param?

Reply:
I kept it for backward-compatibility, but thinking about it again, I think you're right, it brings more confusion. I'll remove it 👍